app/src/main/kotlin/io/homeassistant/companion/android/util/compose/webview/HAWebView.kt

```diff
@@ -0,0 +1,42 @@
+package io.homeassistant.companion.android.util.compose.webview
+
+import android.webkit.WebView
+import android.widget.FrameLayout
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.viewinterop.AndroidView
+
+/**
+ * A composable that displays a WebView.
+ *
+ * This function provides a convenient way to embed a WebView in a Jetpack Compose UI.
+ * It allows for customization of the WebView through the `configure` lambda,
+ * and provides a callback for when the WebView is released via the `onRelease` lambda.
```

Comment:
- `onRelease` no longer seems to exist
 - You're desribing parameters. Why not limit that description to the `@param` description to avoid duplicate (or inconsistent) text? (line 13 is also described in line 17/18)